app/Http/Controllers/AdminController.php

```diff
@@ -6,22 +6,44 @@
 use Auth;
 use Illuminate\Http\Request;
 use Illuminate\Contracts\Pagination\Paginator;
+use Illuminate\Support\Facades\DB;
 
 class AdminController extends Controller
 {
     public function getBasicAdminPage()
     {
+        $cityNumber = DB::table('cities')->count();
+        $countyNumber = DB::table('counties')->count();
+        $stateNumber = DB::table('states')->count();
+
+        //route() to show routes
+        $locationCards = [];
+        $locationCards[] = ["title" => "Cities", "subheading" => "Cities in Counties", "count" => $cityNumber, "manageData" => route("cityView"), "addData" => route("cityAdd")];
+        $locationCards[] = ["title" => "Counties", "subheading" => "Counties in States", "count" => $countyNumber, "manageData" => route("countyView"), "addData" => route("countyAdd")];
+        $locationCards[] = ["title" => "States", "subheading" => "States in the US", "count" => $stateNumber, "manageData" => route("stateView"), "addData" => route("stateAdd")];
+
+        $sources = DB::table('reusenodes')->where("is_source", true)->count();
```

Comment:
Since we're counting all the re-use nodes, could we remove the `->where("is_source", true)` part?